README.md

```diff
@@ -1,11 +1,60 @@
-[![Trace Analytics CI](https://github.com/opensearch-project/trace-analytics/actions/workflows/test-and-build-workflow.yml/badge.svg)](https://github.com/opensearch-project/trace-analytics/actions/workflows/test-and-build-workflow.yml)
-[![codecov](https://codecov.io/gh/opensearch-project/trace-analytics/branch/main/graphs/badge.svg)](https://codecov.io/gh/opensearch-project/trace-analytics)
-
 # OpenSearch Dashboards Observability
 
 The OpenSearch Dashboards Observability plugin has four components: Trace Analytics, Event Analytics, Operational Panels, and Notebooks. Event analytics allows user to use [PPL](https://opensearch.org/docs/latest/search-plugins/ppl/index/) to query indices stored in OpenSearch. Users can also aggregate on their indices to automatically create visualizations, which can be added as operational panels similar to dashboards.
 
-#### Trace Analytics
+## Code Summary
+
+| Type   | Badge        | Module   |
+|--------|--------------|----------|
+| Build  | [![Developer certificate of origin][dco-badge]][dco-badge-link] |  |
+|        | [![Link Checker][link-check-badge]][link-check-link]             |  |
+|        | [![Observability Dashboards CI][dashboard-build-badge]][dashboard-build-link]          | Dashboards-Plugin |
+|        | [![Observability OpenSearch Build CI][opensearch-build-badge]][opensearch-build-link]     | OpenSearch-Plugin  |
+| Unit tests    |    [![codecov][dashboard-codecov-badge]][codecov-link]          | Dashboards-Plugin |
+|        |    [![codecov][opensearch-codecov-badge]][codecov-link]            | OpenSearch-Plugin  |
+| Integration tests    |   [![cypress tests][cypress-test-badge]][cypress-test-link]           | Dashboards-Plugin |
+|        |   [![plugin IT tests][opensearch-it-badge]][opensearch-it-link]          | OpenSearch-Plugin  |
+| Backward compatibility tests    |   [![BWC tests][bwc-tests-badge]][bwc-tests-badge]           | OpenSearch-Plugin  |
+| Issues | [![good first issues open][good-first-badge]][good-first-link]         |          |
+|        | [![bug open][bug-badge]][bug-link]        |          |
+|        | [![ehancement open][ehancement-badge]][enhancement-link] |          |
```

Comment:
plural? also typo on e**n**hancements

do we need "feature" label here? 

FYI these are the labels used by release notes workflow https://github.com/opensearch-project/trace-analytics/blob/84c6fd7830/.github/draft-release-notes-config.yml#L21-L22

Reply:
Oh yes, we should add feature labels here. Do we need any other label here? 
The enhancement label should be singular I guess, as the badge name phrases out as "enhancement issues" https://github.com/ps48/trace-analytics/tree/code_quality#code-summary

Reply:
If the typo was for variable name, I'll change it to plural 👍🏽

Reply:
I think it's the missing "n" in e**n**hancement for open and badge

Reply:
> The enhancement label should be singular I guess, as the badge name phrases out as "enhancement issues"


I meant bug**s** open, enhancement**s** open and not the label name, to be consistent with good first issue**s** open